### PR TITLE
feat: Enroll admin/user and store creds on SecretsManager

### PR DESCRIPTION
**Pull request by vponselvan**
Feature:

    - Enroll admin using the Fabric CA Client SDK and store credentials on SecretsManager
    - Register and enroll users and store credentials on SecretsManager

Update:

    - Add S3 endpoint for accessing the TLS cert
    - Enable private DNS on the VPC endpoint
    - Remove 'register user feature' from README

### CODE REVIEW

File: src/identity-resources.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (lordjabez) [inline src/identity-resources.ts:1]: What do you think about just naming this file `identity.ts` for simplicity and to match the single word pattern of the other source files?

**Review by lordjabez — CHANGES_REQUESTED** [review]
@vponselvan we're getting close here, just a few suggestions.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,6 @@ out a member and its nodes, but the following enhanced functionality
 is planned for future releases:
 
 *  Add support for Java and .NET
-*  Enroll users, storing their credentials in Secrets Manager
```
[the remark this answers is not in the document]
Comment (vponselvan) [reply README.md:14]: Sure, I'll add it. I realized that later yesterday.

File: src/user.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+   * User ID registered with CA
+   */
+  public readonly userId: string;
+
+  /**
+   * User's affiliation to the member
+   */
+  public readonly affiliation: string;
+
+  /**
```
[the remark this answers is not in the document]
Comment (vponselvan) [reply src/user.ts:46]: My bad, I'll fix the property. I added the password cause to create a User object on the fabric sdk, it needs it along with cert and the private key. But, I checked the sdk code right now and it just sets the password to the object and doesn't do anything else with it. So, I'll remove it.

**Comment by lordjabez** [comment]
This branch was merged manually along with some small cleanups.

**Pull request closed without merging**